feature_tests/kotlin/somelib/src/test/kotlin/dev/diplomattest/somelib/CallbackWrapperTest.kt

```diff
@@ -0,0 +1,35 @@
+package dev.diplomattest.somelib
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class CallbackTest {
+    @Test
+    fun testMultiArgCallback() {
+        val cb: (Int ) -> Int = { i -> i + 2 }
+        val calledBack = CallbackWrapper.testMultiArgCallback(cb, 10)
+        assertEquals(22, calledBack)
+    }
+
+    @Test
+    fun testNoArgCallback() {
+        val cb: () -> Unit = { println("Hello this is a callback"); }
+        val calledBack = CallbackWrapper.testNoArgs(cb)
+        assertEquals(-5, calledBack)
+    }
+
+    @Test
+    fun testCbWithStructCallback() {
+        val cb: (CallbackTestingStruct) -> Int = { s -> s.x + s.y}
+        val calledBack = CallbackWrapper.testCbWithStruct(cb)
```

Comment:
@jcrist1  this test is broken with the error `java.lang.IllegalArgumentException: Callback argument class dev.diplomattest.somelib.CallbackTestingStruct requires custom type conversion` -- do you know how to fix this? Is this a bug, or is this a Kotlin feature?